assets/utils/generateMarkdown.js

```diff
@@ -0,0 +1,77 @@
+// TODO: Create a function that returns a license badge based on which license is passed in
+// If there is no license, return an empty string
+function renderLicenseBadge(license) {
+  if (license[0] === "MIT") {
```

Comment:
consider using the switch statement in place of if statements here to help with readability and scalability
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
Ok, thank you!